query.go

```diff
@@ -268,20 +268,20 @@ func MaxBatchSize(maxSize int64) QueryOption {
 // Timeout functional option. The NumRetries option is set to 1 by default
 // unless overridden by the caller.
 func (s *ChainService) queryAllPeers(
-	// queryMsg is the message to broadcast to all peers.
+// queryMsg is the message to broadcast to all peers.
```

Comment:
I think we can remove those now along side the one below?

Reply:
erg soz my formatter does its own thing with these commented parameters 🙃 

we still need the `queryAllPeers` func for the case where we want a certain message to be sent to each of our peers instead of just current best one for things like block header syncing & checking. The work manager doesnt currently support this "send to all" functionality.